util/pace/util/quantity.py

```diff
@@ -476,14 +476,7 @@ def __descriptor__(self) -> "dace.data.Array":
         """The descriptor is a property that dace uses."""
         if dace is None:
             raise ModuleNotFoundError("dace is not installed")
-        storage = (
-            dace.StorageType.GPU_Global
-            if hasattr(self.data, "__cuda_array_interface__")
-            else dace.StorageType.CPU_Heap
-        )
-        return dace.data.Array(
-            dace.typeclass(str(self.data.dtype)), self.shape, storage=storage
-        )
+        dace.data.create_datadescriptor(self.data)
```

Comment:
I just happened to look at this commit by chance, I think it needs a `return` ;)

Reply:
Yes, thanks!